convert-junit4-to-junit5/src/main/java/jb/convert/ConversionOutcome.java

```diff
@@ -0,0 +1,8 @@
+package jb.convert;
+
+public enum ConversionOutcome {
+    Skipped,
+    Unchanged,
+    Converted,
+    Failed
```

Comment:
I personal see enum values like constants and write them in UPPER cases.

Reply:
I do not like when my code screams at me and I do not understand why constants deserve this special treatment. If you feel strongly about this, I can change it.